app/models/concerns/company/california_data_brokers_requestable.rb

```diff
@@ -24,10 +24,12 @@ def update_california_data_brokers
         next if email.blank? || name.blank? || website.blank?
 
         company = Company.find_or_initialize_by(email: email)
-        company.update \
-          category: Company::CATEGORIES[:california_data_broker],
-          name: name,
-          website: website
+        if company.name != name || company.website != website
```

Comment:
I thought Rails prevents hitting the database if there's no changes automatically. I'll have to double check then 😊

Reply:
Oh interesting. I must have misread the situation. I just ran this test as a sanity check and you're right—Rails prevents hitting the database if there are no changes!

```ruby
test "#update doesn't change `updated_at` regardless of new data" do
  company = Company.create \
    category: Company::CATEGORIES[:california_data_broker],
    email: "test@localhost",
    name: "Test Company",
    website: "testcompany.localhost"
  old_updated_at = company.updated_at

  sleep 1
  company.update(email: "test@localhost")

  assert_not_equal old_updated_at, company.reload.updated_at
end
```

Thanks for chipping in here. 🤗